### ISSUE TITLE
Trying to select already-selected topic branch gives error

### ISSUE THREAD

**Issue by greenrd** [magit/magit-topgit#3]
If I try to press Enter on the current topic branch in magit-status, I get the error `GitError! pathspec 't/foo bar' did not match any file(s) known to git.`

**Comment by orivej** [comment]
I can not reproduce this.

**Comment by greenrd** [comment]
I have seen it happening again, but I no longer think it's related to whether the branch is already-selected. It might be related to the length of the branch name.

**Comment by tarsius** [comment]
@greenrd Could you please soon look into this and the other bugs?

**Comment by tarsius** [comment]
@greenrd I don't think it makes much sense to keep this repository here. I cannot help with those issues since I don't use TopGit and also am not particularly interested in learning to use it just so that I can address those issue. Since you are the author and maintainer of TopGit itself, I think it would make more sense to move this repository into your account, next to the TopGit repository. What do you think?

**Comment by greenrd** [comment]
@tarsius Yes, I agree.
